README.md

```diff
@@ -1,6 +1,10 @@
 # Decorator
 
-A collection of ItemDecorations for RecyclerView
+In Android, when you work with RecyclerViews, the standard way of working with paddings and margins is to work with fixed dimensions inside of the layout files.
+
+While this is works for many applications, sometimes the number of required layout files increases too or becomes hard to maintain.
```

Comment:
There's a typo here "this is works". I would also change it to: 
"While this works for many applications, sometimes the margins inside the layout files shouldn't be applied in every screen."